Resources/Prototypes/Magic/projectile_spells.yml

```diff
@@ -51,9 +52,10 @@
   name: Fireball III
   description: The fastest fireball in the west!
   components:
+    - type: LimitedCharges
+    - type: AutoRecharge
+      rechargeDuration: 8
     - type: WorldTargetAction
-      useDelay: 8
-      renewCharges: true
```

Comment:
I forgot to remove renew charges from here. The use delay should still be intact.